README.md

```diff
@@ -27,6 +27,15 @@ ManagementPortal is an application which is used to manage pilot studies for [RA
 - [Documentation](#documentation)
 - [Client libraries](#client-libraries)
 
+## Dependencies
+
+The following are the prerequisites to run ManagementPortal from source on your machine:
+
+- Java 11
+- Gradle (v6 or later is recommended)
```

Comment:
Gradle is not needed, the `./gradlew` command contains all necessary code to run gradle.